crates/re_types/src/datatypes/translation_rotation_scale3d_ext.rs

```diff
@@ -84,9 +84,9 @@ impl TranslationRotationScale3D {
     }
 
     /// Indicate that this transform is from parent to child.
-    /// This is the oppositve of the default, which is from child to parent.
-    #[inline]
+    /// This is the opposite of the default, which is from child to parent.
     #[allow(clippy::wrong_self_convention)]
+    #[inline]
```

Comment:
this case is a bit annoying but didn't want to make the linter needlessly complex